src/client/dfs/dfs.c

```diff
@@ -6850,6 +6856,66 @@ dfs_cont_check(daos_handle_t poh, const char *cont, uint64_t flags, const char *
 		}
 
 		for (i = 0; i < nr_entries; i++) {
+			if (flags & DFS_CHECK_RELINK) {
+				struct dfs_entry	entry = {0};
+				enum daos_otype_t	otype = daos_obj_id2type(oids[i]);
+				char			oid_name[DFS_MAX_NAME + 1];
+				daos_size_t		len;
+				daos_anchor_t		anchor_dir = {0};
+				uint32_t		nr = DFS_ITER_NR;
+				dfs_obj_t		*dir;
+
+				/** Pass 1 - if a file is seen, skip in this pass */
+				if (daos_is_array_type(otype))
+					continue;
+
+				entry.mode = S_IFDIR | 0755;
+				entry.uid = uid;
+				entry.gid = gid;
+				oid_cp(&entry.oid, oids[i]);
+				entry.atime = entry.mtime = entry.ctime = now.tv_sec;
+				entry.atime_nano = entry.mtime_nano = entry.ctime_nano =
+					now.tv_nsec;
+				entry.chunk_size = dfs->attr.da_chunk_size;
+
+				len = sprintf(oid_name, "%"PRIu64".%"PRIu64"",
+					      oids[i].hi, oids[i].lo);
+				D_ASSERT(len <= DFS_MAX_NAME);
+				rc = insert_entry(dfs->layout_v, now_dir->oh, DAOS_TX_NONE,
+						  oid_name, len, DAOS_COND_DKEY_INSERT, &entry);
+				if (rc) {
+					D_ERROR("Failed to insert leaked entry in l+f (%d)\n", rc);
+					D_GOTO(out_lf2, rc);
+				}
+
+				/*
+				 * Ppen the new directory that was relinked and scan it marking the
```

Comment:
Typo in the comment? Ppen --> Open

Reply:
will fix this typo in following PR to not require another CI run.